multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -477,17 +485,20 @@ def fit(
                 stratify=stratify,
                 random_state=np.random.RandomState(seed),
             )
+            train_data = train_data.reset_index(drop=True)
+            tuning_data = tuning_data.reset_index(drop=True)
```

Comment:
1. How about resetting index only for object detection? It seems that only object detection requires this.
2. Consider investigating the reason why we need this, which is necessary to understand our system and avoid fusion potential bugs.

Reply:
Done, also marked a todo to investigate why we need this and remove it.